src/markets.js

```diff
@@ -101,7 +101,7 @@ export async function buyOutcomeTokens() {
     buyTxOpts = Object.assign({}, opts, buyTxOpts)
 
     const market = await this.contracts.Market.at(marketAddress)
-    const collateralToken = await this.contracts.Token.at(
+    const collateralToken = await this.contracts.WETH9.at(
```

Comment:
Is there an IERC20 ABI? It would be more appropriate, as outcome tokens do not support deposit()

Reply:
Thank you @cag  for reviewing it.
I noticed that Token is not being imported anymore on new pm-contracts.
Talking about WETH9, on its ABI taken from pm-contracts build directory, we have:

```
    {
      "constant": false,
      "inputs": [],
      "name": "deposit",
      "outputs": [],
      "payable": true,
      "stateMutability": "payable",
      "type": "function"
    }
```

Other available Token contracts are:
- BasicToken
- OutcomeToken
- OutcomeTokenProxy
- StandardToken

should we go for one of the aforementioned tokens?

Reply:
Whoa, I'm sorry I didn't see this reply earlier! :o

This is a minor issue that it shouldn't get in the way of this getting merged. I'll take a look at pm-contracts packaging and see if there is something to be done there.

Also I've not heard of `BasicToken`...